src/streams/send.rs

```diff
@@ -680,6 +681,28 @@ impl StreamTx {
         //    first ever poll_packet().
         let use_for_padding = self.is_audio.map(|i| !i).unwrap_or(false);
 
+        let mut snapshot = self.send_queue.snapshot(now);
+
+        // Fake some stats for padding.
+        if self.padding > 0 {
+            // TODO: Be more scientific about this factor.
+            const AVERAGE_PADDING_PACKET_SIZE: usize = 200;
```

Comment:
At the very least I think we can make this higher. Even at 0.5Mbit/s the padding burst will be ~3kb and as such we'll probably use a lot of fairly large packets for spurious resends. 700-900 seems like a better first guess than 200

Reply:
Ok. I change that.